fractions/microprofile/microprofile-opentracing/pom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.wildfly.swarm</groupId>
+    <artifactId>build-parent</artifactId>
+    <version>2018.1.0-SNAPSHOT</version>
+    <relativePath>../../../build-parent/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>microprofile-opentracing</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Microprofile-OpenTracing</name>
+  <description>OpenTracing integration for Microprofile</description>
+
+  <properties>
+    <swarm.fraction.stability>experimental</swarm.fraction.stability>
+    <swarm.fraction.tags>Microprofile, OpenTracing, Monitoring</swarm.fraction.tags>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>spi</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>meta-spi</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax</groupId>
+      <artifactId>javaee-api</artifactId>
+      <version>7.0</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>container</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>jaxrs</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>cdi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>microprofile</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
```

Comment:
Remove `wildfly-jaxrs`, `microprofile`and `javaee-api` as dependencies to the fraction

Reply:
When I remove dependency on `microprofile` TCK fails on 


```
java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory
	at org.eclipse.microprofile.opentracing.tck.OpentracingClientTests.clearTracer(OpentracingClientTests.java:1075)
	at org.eclipse.microprofile.opentracing.tck.OpentracingClientTests.beforeEachTest(OpentracingClientTests.java:116)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.logging.LogFactory
	at org.eclipse.microprofile.opentracing.tck.OpentracingClientTests.clearTracer(OpentracingClientTests.java:1075)
	at org.eclipse.microprofile.opentracing.tck.OpentracingClientTests.beforeEachTest(OpentracingClientTests.java:116)
```

Reply:
So `commons-logging` or the like needs to be a dependency, maybe. And it looks like it's needed for the test more than the actual fraction.

`microprofile` refers to the entire collection of MP fractions in WF Swarm, so should not be used within a single MP fraction otherwise every other MP fraction will be included in open tracing

Reply:
I solved it, I have added commons-logging to the test module